### ISSUE TITLE
Breizorro not working

### ISSUE THREAD

**Issue by ktrehaeven** [caracal-pipeline/caracal#1497]
Hi

I'm trying to use breizorro to generate clean masks but I get the following error attached. Perhaps I'm not using the option correctly. Could someone please advise on how exactly it works?

My thinking was that you generate a deep wsclean automasked image first solely to generate the breizorro clean image, reimage with this mask and then start calibrating from there. But whatever I try I cant seem to make a breizorro mask inside caracal
[log-caracal.txt](https://github.com/caracal-pipeline/caracal/files/11189710/log-caracal.txt)

**Comment by o-smirnov** [comment]
Probably just this, check your quota/disk space:

> OSError: [Errno 28] No space left on device

For future reference, these errors should go to Stimela 1 (https://github.com/ratt-ru/Stimela) since that's what Caracal uses. This repo is for Stimela2 which is a completely different package. @SpheMakh it was your idea to drop the "2" from the repo name, perhaps we should re-introduce it to avoid this sort of confusion?

**Comment by o-smirnov** [comment]
Oh wait, my bad, you posted on the caracal repo (as you should) not stimela, so forget what I said...

**Comment by ktrehaeven** [comment]
Apparently, I still have over 500Gb of storage space left on the machine
![Screenshot from 2023-04-10 12-10-51](https://user-images.githubusercontent.com/77777044/230881918-3397b5e3-d5f1-40bc-89ab-37e5aec4205f.png)

So idk

**Comment by ktrehaeven** [comment]
@o-smirnov  surely 500Gb space should be enough though

**Comment by o-smirnov** [comment]
IDK, how big is the image? What filesystem is your output directory on? It definitely falls over when trying to make a copy of the input into the output image... if so, maybe someone else needs to take a look at how it's being invoked?

**Comment by ktrehaeven** [comment]
the restored image is 250Mb. the output dir is in /net/janis/home/trehaeven/MACSJ2140.2-2339_UHF/output

@Athanaseus perhaps you can help here.

**Comment by KshitijT** [comment]
Reopening this issue as I have encountered this on ilifu(the disk space is sufficient for further processing involving multiple images so I can't imagine what's happening here) and it has been also encountered by Konstantinos on ilifu.

**Comment by Athanaseus** [comment]
Please confirm your `stimela` release.
It should be fixed in version `1.7.7` via https://github.com/ratt-ru/Stimela/pull/779.

**Comment by Athanaseus** [comment]
@KshitijT, did it finally work for you?

**Comment by KshitijT** [comment]
I don't think it did @Athanaseus , but let me try again with the latest version and get back to you.

**Comment by KshitijT** [comment]
Seems to work now, closing this issue.
